projects/plugins/jetpack/extensions/blocks/blogroll/use-site-recommendations.js

```diff
@@ -0,0 +1,79 @@
+import { store as coreStore, useEntityProp } from '@wordpress/core-data';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect, useState, useMemo } from '@wordpress/element';
+import { debounce } from '../../shared/debounce';
+import { createBlockFromRecommendation } from './utils';
+
+export function useBlogrollItemAttributes( clientId ) {
```

Comment:
This should be plural I think:

```suggestion
export function useBlogrollItemsAttributes( clientId ) {
```